dask/dataframe/core.py

```diff
@@ -4148,6 +4148,11 @@ def itertuples(self, index=True, name="Pandas"):
             for row in df.itertuples(index=index, name=name):
                 yield row
 
+    @derived_from(pd.DataFrame)
+    def items(self):
+        for key in self.columns:
+            yield key, self[key]
```

Comment:
Tiny comment: maybe use `label` and `content` like pandas does? 

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.items.html